src/server/controllers/verification/session-state/session-state.handlers.ts

```diff
@@ -40,6 +40,7 @@ export async function addInputFilesEndpoint(req: Request, res: Response) {
     return { path: pb.path, content: pb.buffer.toString(FILE_ENCODING) };
   });
 
+  const dryRun = Boolean(req.body.dryRun)
```

Comment:
Is this case-insensitive? Should we use the same casing when retrieving parameters?

Reply:
Yes, and this must be _req.request.dryrun_ in fact.